tests/helpers/runif.py

```diff
@@ -66,6 +87,11 @@ def __new__(
             conditions.append(torch_version < LooseVersion(min_torch))
             reasons.append(f"torch>={min_torch}")
 
+        if min_python:
+            py_version = f"{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}"
+            conditions.append(py_version < LooseVersion(min_python))
+            reasons.append(f"python>={min_python}")
```

Comment:
```suggestion
            reasons.append(f"requires python>={min_python}")
```

Reply:
requires is already in the group test `Requires: [...]`